app/uk/gov/hmrc/nisp/config/ApplicationConfig.scala

```diff
@@ -75,4 +81,39 @@ object ApplicationConfig extends ApplicationConfig with ServicesConfig {
   override val ivUpliftUrl: String = configuration.getString(s"identity-verification-uplift.host").getOrElse("")
   override val ggSignInUrl: String = configuration.getString(s"government-gateway-sign-in.host").getOrElse("")
   override val twoFactorUrl: String = configuration.getString(s"two-factor.host").getOrElse("")
+  override lazy val breadcrumbServiceUrl: String = configuration.getString(s"breadcrumb-service.url").getOrElse("")
+
+  override lazy val perTaxFrontEndUrl: String  = configuration.getString(s"pertax-frontend.url").getOrElse("")
+  lazy val pertaxServiceUrl = s"$perTaxFrontEndUrl/"
+  val initialBreadCrumbList = List((URLEncoder.encode(Messages("nisp.breadcrumb.account"),"UTF-8"), pertaxServiceUrl))
+  lazy val mainContentHeaderPartialUrl = s"$breadcrumbServiceUrl/integration/main-content-header"
+
+  private[config] def buildBreadCrumb(request: Request[_]): List[(String, String)] = {
```

Comment:
Could we use a case class rather than a tuple. The logic around this code is a little long to use a tuple.